android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt

```diff
@@ -115,6 +116,7 @@ class VPNService : android.net.VpnService() {
         // is bound to this service, it will request that the main app broadcast
         // the user provided preference for this. So it should not be long until
         // the correct value is set here. See VPNServiceBinder > onTransact > ACTIONS.registerEventListener.
+        // Lol it' so broken.
```

Comment:
How is this broken?

Reply:
OOps that was before i escalated with you q__q"